src/Detail/Blitline/Client/ServiceDescription/Blitline.php

```diff
@@ -31,9 +31,9 @@
             'documentationUrl' => 'http://www.blitline.com/docs/api',
             'parameters'       => array(
                 'src' => array(
-                    'description' => 'The location of the image you wish to process',
+                    'description' => 'The S3 location of the image you wish to process',
```

Comment:
Is it an undocumented Blitline feature to pass an array? The documentation (http://www.blitline.com/docs/api) does not mention it.